### PR TITLE
[15.0][IMP] bi_sql_editor: create filters for date fields.

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Date filters provide default options by serveral periods. We want to take advantage of this.

image

cc @LoisRForgeFlow @JoanSForgeFlow @ForgeFlow

### CODE REVIEW

File: bi_sql_editor/models/bi_sql_view_field.py [hunk 1]
```diff
@@ -250,12 +250,20 @@ def _prepare_search_field(self):
...
+        field_filter = ""
         if self.field_description and self.is_group_by:
-            res = """<filter name="group_by_%s" string="%s"
+            group_by_filter = """<filter name="group_by_%s" string="%s"
                         context="{'group_by':'%s'}"/>\n""" % (
                 self.name,
                 self.field_description,
                 self.name,
             )
+        if self.field_description and self.ttype in ["date", "datetime"]:
```
Comment (legalsylvain) [inline bi_sql_editor/models/bi_sql_view_field.py:262]: I don't

**Review by legalsylvain — APPROVED** [review]
Thanks ! LGTM.

**Review by LoisRForgeFlow — APPROVED** [review]
Looks like a good improvement :+1:

**Comment by legalsylvain** [comment]
/ocabot merge patch

@JordiBForgeFlow feel free to port this patch upstream. (16 /17)

**Comment by LoisRForgeFlow** [comment]
@JordiBForgeFlow Should this be forward ported?

**Pull request merged**